UniAtHome/UniAtHome.BLL/Services/EmailService.cs

```diff
@@ -0,0 +1,64 @@
+﻿using MailKit.Net.Smtp;
+using Microsoft.Extensions.Configuration;
+using MimeKit;
+using System.Threading.Tasks;
+using UniAtHome.BLL.Interfaces;
+
+namespace UniAtHome.BLL.Services
+{
+    public sealed class EmailService : IEmailService
+    {
+        private EmailOptions options;
+
+        public EmailService(IConfiguration configuration)
+        {
+            options = new EmailOptions();
+            configuration.Bind("Email", options);
```

Comment:
why are u binding your config here? it's usually done in startup and send via IOptions